src/Adapter/Product/AdminProductDataProvider.php

```diff
@@ -80,19 +83,32 @@ public function __construct(EntityManager $entityManager, ImageManager $imageMan
     public function getPersistedFilterParameters()
     {
         $employee = Context::getContext()->employee;
-        $shop = Context::getContext()->shop;
-        $filter = $this->entityManager->getRepository('PrestaShopBundle:AdminFilter')->findOneBy(array(
-            'employee' => $employee->id ?: 0,
-            'shop' => $shop->id ?: 0,
-            'controller' => 'ProductController',
-            'action' => 'catalogAction',
-        ));
-        /* @var $filter AdminFilter */
-        if (!$filter) {
-            return AdminFilter::getProductCatalogEmptyFilter();
+        $employeeId = $employee->id ?: 0;
```

Comment:
For people wondering what is the compatibility for a shorthand ternary test: PHP 5.3 required.

Reply:
not in develop branch my friend, we are using PHP 5.6